scripts/load-shakespeare.sh

```diff
@@ -41,10 +41,11 @@ curl "http://localhost:9200/shakespeare" \
   -X PUT \
   -d @"$PROJECT_ROOT"/elasticsearch/data/mappings.json
 
+echo " "
 echo "Loading the data into the Elasticsearch cluster"
-curl "http://localhost:9200/shakespeare/doc/_bulk" \
-   -s \
-   -H "Content-Type: application/x-ndjson" \
-   -X POST \
+
+curl "http://localhost:9200/shakespeare/doc/_bulk?pretty" \
+    -H "Content-Type: application/x-ndjson" \
+   --retry 5 \
```

Comment:
Why do we need this?

Reply:
```
curl "http://localhost:9200/shakespeare/doc/_bulk?pretty" \
    -H "Content-Type: application/x-ndjson" \
   --retry 5 \
   --data-binary @"$PROJECT_ROOT"/elasticsearch/data/shakespeare_6.0.json
```

This `curl` command makes a POST request to upload some data to populate the index mapping that is created earlier in this file. This data is used validate that the cluster is working as intended, e.g the ingest nodes are reachable and that the cluster indexes are sharded across all of the nodes.

Reply:
Right. Was not clear. Why do we need the retry?

Reply:
Let’s merge this beast of a PR and see if we can remove the retry